docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -59,22 +59,24 @@ to create a new trusted application, go to **Manage** -> **Trusted applications*
 that looks for static attributes to determine if a file is malicious or benign.
 
 By default, malware protection is enabled on Windows, macOS, and Linux hosts.
-To disable malware protection, switch the **Malware protections enabled** toggle off.
+To disable malware protection, turn off the **Malware protections** toggle.
 
 Malware protection levels are:
 
 * **Detect**: Detects malware on the host and generates an alert. The agent will **not** block malware.
 You must pay attention to and analyze any malware alerts that are generated.
 * **Prevent** (Default): Detects malware on the host, blocks it from executing, and generates an alert.
 
+These additional options are available for malware protection:
+
+* **Blocklist**: Enable or disable the <<blocklist,blocklist>> for all hosts associated with this {elastic-defend} policy. The blocklist allows you to prevent specified applications from running on hosts, extending the list of processes that {elastic-defend} considers malicious.
+
+* **Scan files upon modification**: By default, {elastic-defend} scans files every time they're modified, which can be resource-intensive for the host. Turn off this option to scan files only when they're executed. This means that malware will only be identified when it attempts to run, but it still provides a robust level of protection while improving endpoint performance.
```

Comment:
💯  correct, but consider something like to bring attention to the trigger:

```suggestion
* **Scan files upon modification**: By default, {elastic-defend} scans files every time they're modified, which can be resource-intensive on hosts where files are frequently modified such as servers and developer machines. Turn off this option to scan files only when they're executed. {elastic-defend} will continue to identify malware as it attempts to run, providing a robust level of protection while improving endpoint performance.
```

I also removed the second use of "it" which conflicted with the first.